server/branch.ts

```diff
@@ -0,0 +1,257 @@
+import * as path from "path";
+import * as ajv from "ajv";
+import * as moment from "moment-timezone";
+import { QuestionBranchConfig, QuestionBranchSettings, IUser } from "./schema";
+import { config, readFileAsync, renderMarkdown } from "./common";
+import { QuestionBranches, Questions, TextBlocks } from "./config/questions.schema";
+
+// tslint:disable:interface-name variable-name
+interface Labels {
+	[questionName: string]: string;
+}
+interface QuestionBranchTypes {
+	"Application": ApplicationBranch;
+	"Confirmation": ConfirmationBranch;
+	"Noop": NoopBranch;
+}
+type QuestionBranch = QuestionBranchTypes[keyof QuestionBranchTypes];
+
+const SCHEMA_FILE: string = "./config/questions.schema.json";
+
+export class NoopBranch {
+	public readonly name: string;
+	public readonly type: keyof QuestionBranchTypes = "Noop";
+
+	public textBlocks: TextBlocks | undefined;
+	public questions: Questions;
+	public questionLabels: Labels;
+
+	protected readonly location: string;
```

Comment:
change to `schemaPath` or similar

Reply:
This is actually the location of the `questions.json` file on-disk. It's there for preliminary support for multiple `questions.json` files